Assets/MixedRealityToolkit/_Core/Resources/Shaders/InvisibleShader.shader

```diff
@@ -0,0 +1,54 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+Shader "MixedRealityToolkit/InvisibleShader" {
```

Comment:
Can this not be achieved with the Standard shader?  Do we really need more that one?
@Cameron-Micka ?